core/src/main/java/org/kiwix/kiwixmobile/core/CoreApp.java

```diff
@@ -64,6 +65,9 @@ public static void setCoreComponent(CoreComponent appComponent) {
   @Override
   protected void attachBaseContext(Context base) {
     super.attachBaseContext(base);
+    if (BuildConfig.DEBUG) {
+      MultiDex.install(this);
+    }
```

Comment:
Why are we using MultiDex only for debug?

Reply:
Because our release version is Proguarded (R8-ed) and our method count post minification is well below the 65K limit.
Debug is not proguarded so it is does need to be multidexed.

If we don't need to do it I don't think we should be doing it, it complicates the install process

Reply:
Makes sense. This can be added as a comment to these lines. :p